woocommerce/woocommerce-gateway-stripe
# POC: fix tests

# Changes proposed in this Pull Request:

Issue: Link to the GitHub issue this PR addresses (if appropriate).
Fixes #

Description: Write a brief summary about this PR. As you compose your summary, consider each of these questions and address them if appropriate. Why is this change needed? What does this change do? Were there other solutions you considered? Why did you choose to pursue this solution? Describe any trade-offs you might have had to make.

Images or gifs: Include before and after screenshots or gifs/videos when it makes sense.


# Testing instructions

Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?

(remove me) Please follow the following guidelines when writing testing instructions:

- Include screenshots if there is no similar flow in the [Critical flows](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Critical-flows).
- Assume instructions will be copied over to the [Release Testing Instructions wiki page](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Release-Testing-Instructions).
- Assume instructions will be followed by external testers.
- Assume tester does not have intimate knowledge of Stripe.

-------------------

(remove me)
